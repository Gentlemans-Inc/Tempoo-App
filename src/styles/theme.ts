const theme = {
  colors: {
    almostWhite: '#f1f1f1',
    error: '#FF453A',
    lightGray: '#d1d1d6',
    mediumGray: '#8E8E98',
    gray: '#72788A',
    placeholder: '#D1D1D6',
    primary: '#65AFEC',
    textDark: '#343434',
  },
};

export default theme;
