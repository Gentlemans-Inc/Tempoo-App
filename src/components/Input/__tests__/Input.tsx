import React from 'react';
import renderer from 'react-test-renderer';
import {Input} from '../index';
import theme from '../../../styles/theme';
import 'jest-styled-components';

test('Input Component', () => {
  const tree = renderer.create(<Input theme={theme} />).toJSON();
  expect(tree).toMatchSnapshot();
});
