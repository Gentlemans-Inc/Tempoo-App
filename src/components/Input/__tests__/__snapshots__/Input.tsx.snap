// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input Component 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#fff",
        "borderColor": "#72788A",
        "borderRadius": 8,
        "borderWidth": 0.1,
        "flexDirection": "row",
        "marginBottom": 20,
        "paddingLeft": 22.5,
        "paddingRight": 22.5,
        "width": "85%",
      },
    ]
  }
  theme={
    Object {
      "colors": Object {
        "almostWhite": "#f1f1f1",
        "error": "#FF453A",
        "gray": "#72788A",
        "lightGray": "#d1d1d6",
        "mediumGray": "#8E8E98",
        "placeholder": "#D1D1D6",
        "primary": "#65AFEC",
        "textDark": "#343434",
      },
    }
  }
>
  <TextInput
    allowFontScaling={true}
    rejectResponderTermination={true}
    style={
      Array [
        Object {
          "color": "#343434",
          "marginLeft": 11.5,
          "paddingBottom": 6.5,
          "paddingLeft": 6.5,
          "paddingRight": 6.5,
          "paddingTop": 6.5,
          "width": "90%",
        },
      ]
    }
    theme={
      Object {
        "colors": Object {
          "almostWhite": "#f1f1f1",
          "error": "#FF453A",
          "gray": "#72788A",
          "lightGray": "#d1d1d6",
          "mediumGray": "#8E8E98",
          "placeholder": "#D1D1D6",
          "primary": "#65AFEC",
          "textDark": "#343434",
        },
      }
    }
    underlineColorAndroid="transparent"
  />
</View>
`;
